src/Microsoft.Framework.Runtime/DependencyManagement/LockFileFormat.cs

```diff
@@ -71,7 +82,7 @@ private LockFile ReadLockFile(JObject cursor)
         {
             var lockFile = new LockFile();
             lockFile.Islocked = ReadBool(cursor, "locked", defaultValue: false);
-            lockFile.Version = ReadInt(cursor, "version", defaultValue: int.MaxValue);
+            lockFile.Version = ReadInt(cursor, "version", defaultValue: int.MinValue);
```

Comment:
Version int.MinValue is always out-of-date. We use this way to mark a lockfile as invalid.